shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -4352,5 +4352,49 @@ TEST_F(EmbedderTest, CanLaunchAndShutdownWithAValidElfSource) {
   engine.reset();
 }
 
+TEST_F(EmbedderTest, FrameInfoContainsValidWidthAndHeight) {
```

Comment:
Another test here to verify that engine initialization is aborted if both the old and new callbacks are specified by the embedder.